src/Symfony/Component/Templating/TemplateReference.php

```diff
@@ -75,7 +75,7 @@ public function all()
      */
     public function getPath()
     {
-        return $this->parameters['name'];
+        return 'views/'.$this->parameters['name'];
```

Comment:
Allright. I used it because same logic is applied in frameworkbundle. But I see what you mean.